pkg/ebpf/events_enrich.go

```diff
@@ -78,7 +78,7 @@ func (t *Tracee) enrichContainerEvents(ctx gocontext.Context, in <-chan *trace.E
 			case event := <-in:
 				eventID := events.ID(event.EventID)
 				// send out irrelevant events (non container or already enriched), don't skip the cgroup lifecycle events
-				if (event.ContainerID == "" || event.ContainerImage != "") && eventID != events.CgroupMkdir && eventID != events.CgroupRmdir {
+				if (event.Container.ID == "" || event.Container.Name != "") && eventID != events.CgroupMkdir && eventID != events.CgroupRmdir {
```

Comment:
`ContainerImage` -> `Container.Name`
is that correct or should have it been `Container.ImageName`?

Reply:
@rafaeldtinoco got it right.
As for the name changes:
`ContainerName` -> `Container.Name`
`ContainerImage` -> `Container.ImageName`